.github/workflows/static_analysis.yml

```diff
@@ -0,0 +1,27 @@
+name: Static analysis
+on: [push, pull_request]
+
+jobs:
+  Cpp:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Formatting
+      uses: Flamefire/clang-format-lint-action@master
```

Comment:
Will this change to something local?

Reply:
This is the upstream one: https://github.com/DoozyX/clang-format-lint-action

I used my fork as the upstream (was) missing a PR of mine and I'm waiting for a tag now. So I'd rather change it to that one ones it is tagged.

Reply:
binary blobs for a code formatter ... what could possibly go wrong. Anyway, it looks like GitHub actions are designed this way, so that's fine.